haya-hayaodeh/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -10,3 +10,11 @@
   conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
 */
 
+for (let i = 99; i>0 ; i--) {
+  
+ console.log(i + " bottles of beer on the wall, " + i + " bottles of beer.");
+ console.log("Take one down and pass it around, "+ (i-1) + " bottles of beer on the wall.")
+console.log (" ")
+  
+}
+ console.log ("No more bottles of beer on the wall, no more bottles of beer.Go to the store and buy some more, 99 bottles of beer on the wall.")
```

Comment:
It's clever of you to move the final log statement outside of the for loop. Does exactly what it needs to do with a minimum of code.